library/api/library.api

```diff
@@ -49,20 +47,3 @@ public final class com/transferwise/sequencelayout/SequenceStep : android/widget
 	public final fun setTitleTextAppearance (I)V
 }
 
-public abstract interface class com/transferwise/sequencelayout/SequenceStep$OnStepChangedListener {
-	public abstract fun onStepChanged ()V
-}
-
-public final class com/transferwise/sequencelayout/SequenceStepDot : android/widget/FrameLayout {
-	public field _$_findViewCache Ljava/util/Map;
-	public fun <init> (Landroid/content/Context;)V
-	public fun <init> (Landroid/content/Context;Landroid/util/AttributeSet;)V
-	public fun <init> (Landroid/content/Context;Landroid/util/AttributeSet;I)V
-	public fun _$_clearFindViewByIdCache ()V
-	public fun _$_findCachedViewById (I)Landroid/view/View;
-	public fun setActivated (Z)V
-	public final fun setDotBackground$com_transferwise_sequencelayout_null_release (II)V
-	public fun setEnabled (Z)V
-	public final fun setPulseColor$com_transferwise_sequencelayout_null_release (I)V
-}
-
```

Comment:
These `internal` ones should not have been here in the first place at all, but after we bumped from Kotlin 1.4.x -> 1.6 these got generated incorrectly with 0.5.0 version of the plugin. Now that I have updated to 0.8.0 which includes the fix to this, these are free to remove as none of them are a public API.

See: https://github.com/Kotlin/binary-compatibility-validator/issues/53